pkg/api/validation/register.go

```diff
@@ -1,6 +1,8 @@
 package validation
 
 import (
+	_ "github.com/openshift/origin/pkg/api/install"
```

Comment:
Oh, it's because it has to be run before `init` is invoked.  You don't get ordering guarantees on when imports are available within the package.